pkgs/by-name/as/ashell/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+  makeWrapper,
+  pkg-config,
+  libxkbcommon,
+  libGL,
+  pipewire,
+  libpulseaudio,
+  wayland,
+  vulkan-loader,
+  mesa,
+}:
+let
+  libs = [
+    libpulseaudio
+    wayland
+    mesa.drivers
+    vulkan-loader
+    libGL
+  ];
+in
+rustPlatform.buildRustPackage rec {
+  pname = "ashell";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "MalpenZibo";
+    repo = "ashell";
+    rev = version;
```

Comment:
`tag = `?

Reply:
you're right! Sorry, I was kinda mindlessly copying from [bat](https://github.com/NixOS/nixpkgs/blob/nixos-24.11/pkgs/by-name/ba/bat/package.nix#L81).